doc/releases/release-notes-3.6.rst

```diff
@@ -147,6 +147,11 @@ Build system and infrastructure
 
 * Fixed an issue whereby using some shields with sysbuild would cause a cmake Kconfig error.
 
+* Added support for sysbuild and zephyr modules to set a signing script (``SIGNING_SCRIPT`` in
+  sysbuild, ``SIGNING_SCRIPT`` of ``zephyr_property_target`` for zephyr modules) to set a cmake
+  file which is used for signing. The priority is now sysbuild, zephyr propety target, MCUboot
+  default fallback (if enabled).
```

Comment:
Setting the signing script via `zephyr_property_target` is not new.
Maybe just mention the new variable and its precedence compared to the existing property.

Other errors:
* Capitalization of Zephyr and CMake
* "to set a signing script to set a cmake file which is used for signing"
* "propety"

Reply:
Updated